lms/lib/comment_client/user.py

```diff
@@ -180,6 +185,15 @@ def retire(self, retired_username):
             metric_tags=self._metric_tags
         )
 
+    def delete(self):
```

Comment:
I took another look at this method, and the naming seems a bit misleading. We don't actually delete anything but run custom logic which is called before actual user removal (as far as I see). So maybe `update_username` or smth like that would be more suitable... What are your thoughts? @dyudyunov

Reply:
Agree with You, thanks

Reply:
Waiting for change.

Reply:
@dyudyunov change method naming or add an appropriate docstring please.